index.html

```diff
@@ -0,0 +1,208 @@
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <!--Header section-->
+    <header>
+    <!--Navigation mobile-->
+        <nav class="menu_mobile">
+            <div class="logo">
+                <a href="#">Lesapiti</a>
+            </div>
+            <div>
+                <i class="fa fa-bars"></i>
+            
+            </div>
+        </nav>
+    <!--Navigation desktop-->
+        <nav class="menu_desktop">
+            <div class="logo">
+                <a href="#">Lesapiti</a>
+            </div>
+            <div>
+                <ul class="nav_menu">
+                    <li> <a href="#">Portfolio</a></li>
+                    <li> <a href="#">About</a></li>
+                    <li> <a href="#">Contact</a></li>
+                </ul>
+            </div>
+        </nav>
+        </header>
+    
+    <!--Home section-->
+    <section class="home">
+        <div class="home-content">
+            <h1>I'm Petro <br> Glad to see you!</h1>
+            <p class="my-intro">I'm a software developer! I can help you build a product ,
+                 feature or website Look through some of my work and experience!
+                  If you like what you see and have a project you need coded, don't 
+                  hestiate to contact me.
+            </p>
+            <br>
+            <h4 class="lets-connect-message">LET'S CONNECT</h4>
+            <ul class="connect-media">
+                <li><a href="#"><img src="images/twitter.svg" alt="twitter"></a></li>
+                <li><a href="#"><img src="images/Linkedinicon.svg" alt="linkedin"></a></li>
+                <li><a href="#"><img src="images/medium.svg" alt="medium"></a></li>
+                <li><a href="#"><img src="images/github.svg" alt="github"></a></li>
+                <li><a href="#"><img src="images/angellist.svg" alt="angellist"></a></li>
+            </ul>
+        </div>
+    </section>
+    <!--Work section-->
+    <section class="work">
+        <!---The first card -->
+        <div class="flex">
+            <div class="flex_image">
+                <img class="mobile_image" src="images/card1-image.png" alt="Image for Card 1">
+                <img class="desktop_image" src="images/card1.png" alt="Image for desktop card 1">
+            </div>
+            <div class="flex_content">
+                <h2>TONIC</h2>
+                <div class="canopy">
+                    <h3>CANOPY</h3>
+                    <ul class="canopy-list"><li>BACK&nbsp;END&nbsp;Dev</li><li>2015</li></ul>
+                </div> 
+                <p class="card-info">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="list second">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+                </ul>
+                <button class="see-more">See Project</button>
```

Comment:
- [ ] It appears that the new changes that were introduced broke the design for your buttons. Right now your buttons have an overflow and the text does not fit within the button. Kindly fix this by reducing the size of the button text and the project will be ready for approval.

**This is what your buttons look like:**
![image](https://user-images.githubusercontent.com/86318284/178032333-f3735c81-8351-4070-8a6a-cdbb3cc6e0f2.png)

**This is how the buttons should look like:**
![image](https://user-images.githubusercontent.com/86318284/178032484-3e056ea4-2dbd-4aad-ba0e-b530b859ff52.png)